packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -111,8 +114,8 @@ export async function produceBlockBody<T extends BlockType>(
     );
   }
 
-  const forkName = currentState.config.getForkName(blockSlot);
-  if (forkName !== ForkName.phase0 && forkName !== ForkName.altair) {
+  const forkInfo = currentState.config.getForkInfo(blockSlot);
+  if (forkInfo.name !== ForkName.phase0 && forkInfo.name !== ForkName.altair) {
```

Comment:
```suggestion
  if (fork !== ForkName.phase0 && fork !== ForkName.altair) {
```

Then everywhere you were passing the `seq: ForkSeq` change the syntax to `ForkSeq[fork] > ForkSeq.altair` so we only use `fork: ForkName` everywhere